lib/std/hash/RapidHash.zig

```diff
@@ -0,0 +1,125 @@
+const std = @import("std");
+
+const readInt = std.mem.readInt;
+const assert = std.debug.assert;
+const expect = std.testing.expect;
+const expectEqual = std.testing.expectEqual;
+
+const RAPID_SEED: u64 = 0xbdd89aa982704029;
+const RAPID_SECRET: [3]u64 = .{ 0x2d358dccaa6c78a5, 0x8bb84b93962eacc9, 0x4b33a62ed433d4a3 };
+
+pub fn hash(seed: u64, input: []const u8) u64 {
+    const sc = RAPID_SECRET;
+    const len = input.len;
+    var a: u64 = 0;
+    var b: u64 = 0;
+    var k = input;
+    var is: [3]u64 = .{ seed, 0, 0 };
+
+    is[0] ^= mix(seed ^ sc[0], sc[1]) ^ len;
+
+    if (len <= 16) {
+        if (len >= 4) {
+            const d: u64 = ((len & 24) >> @intCast(len >> 3));
+            const e = len - 4;
+            a = (r32(k) << 32) | r32(k[e..]);
+            b = ((r32(k[d..]) << 32) | r32(k[(e - d)..]));
+        } else if (len > 0)
```

Comment:
Could you put this `if` case into brackets, I think it'd be more readable that way.